qiskit/providers/ibmq/providerbackends.py

```diff
@@ -16,15 +16,17 @@
 
 import re
 import keyword
+from typing import Iterable
 from types import SimpleNamespace
 
+from .ibmqbackend import IBMQBackend
 from .api_v2.exceptions import RequestsApiError
 
 
 class ProviderBackends(SimpleNamespace):
     """Backend namespace for an IBM Quantum Experience account provider."""
 
-    def __init__(self, provider):
+    def __init__(self, provider: 'AccountProvider') -> None:
```

Comment:
Temporary forward reference marker. Needs to be dealt with after addressing the circular dependency that arises when the class is imported.